tests/utils/googleAnalytics/event.test.ts

```diff
@@ -0,0 +1,46 @@
+import * as faker from 'faker';
+
+import * as initUtils from '../../../src/utils/googleAnalytics/initialize';
+import {event} from '../../../src/utils/googleAnalytics/event';
+
+describe('googleAnalytics.initialize', () => {
+  const setUp = () => {
+    const trackingId = faker.lorem.word();
+    initUtils.initialize(trackingId);
+
+    const mockDate = faker.datatype.datetime();
+    jest.useFakeTimers();
+    jest.setSystemTime(mockDate);
+
+    const name = faker.lorem.word();
+    const params = {foo: 'bar'};
+
+    const gtagSpy = jest.spyOn(initUtils, 'gtag');
+    const consoleInfoSpy = jest.spyOn(global.console, 'info');
+
+    return {
+      name,
+      params,
+      gtagSpy,
+      consoleInfoSpy,
+    };
+  };
+
+  test('should record a event with fake name', () => {
+    const {name, gtagSpy, consoleInfoSpy} = setUp();
+
+    event(name);
+
+    expect(gtagSpy).toHaveBeenNthCalledWith(1, 'event', name, undefined);
```

Comment:
```suggestion
    expect(gtagSpy).toHaveBeenCalledWith('event', name, undefined);
```

1번만 호출할 땐 횟수 생략해줘도 될 것 같습니다 ㅎㅎ

Reply:
오호! 감사합니다!